packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -46,5 +46,9 @@ export const DefaultPalette: IPalette = {
   tealLight: '#00b294',
   greenDark: '#004b1c',
   green: '#107c10',
-  greenLight: '#bad80a'
+  greenLight: '#bad80a',
+  contrastBlackDisabled: '#00ff00',
```

Comment:
Yes, greyText maps to the "disabled" system color it works in all high contrast modes. Note, that there's a lot more subtlety to high contrast outside of white on black and black on white. If you bring up the high contrast settings dialog in the OS you'll see you can choose what the disabled text looks like. It'll look something like this: 
![image](https://user-images.githubusercontent.com/7033561/28193038-fc5ccdf8-67ee-11e7-879f-775dc3c2bed9.png).

We should shy away from hardcoding values when we can just as easily match the user's actual system colors "for free"